### ISSUE TITLE
Add option to override Private Windows' proxies

### ISSUE THREAD

**Issue by qwerty12** [bekh6ex/firefox-container-proxy#38]
Thanks for this extension. Could an option be added to override the proxy used by Private Windows too? I've tried the following here and it works (sorry for the lack of PR, but this lacks i18n and an icon):

```diff
diff --git a/src/options/ContainerListView.ts b/src/options/ContainerListView.ts
index e31a175..1eb7109 100644
--- a/src/options/ContainerListView.ts
+++ b/src/options/ContainerListView.ts
@@ -46,7 +46,15 @@ export class ContainerListView implements Component {
       icon: '',
       iconUrl: ''
     })
-    return m('.containers', [...items, defaultContainer])
+    const privateContainer = this.renderContainerItem({
+      cookieStoreId: 'firefox-private',
+      name: 'Private Browsing',
+      color: '',
+      colorCode: '',
+      icon: '',
+      iconUrl: ''
+    })
+    return m('.containers', [...items, defaultContainer, privateContainer])
   }
 
   renderSelectProxy (cookieStoreId: string, proxyId: string): Vnode {
```

**Comment by ntninja** [comment]
This works! Please add this!

**Comment by bekh6ex** [comment]
Implemented in version 0.1.19.
Thank you for the code!

**Comment by ntninja** [comment]
Thanks! :-)
